package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fauna-shell",
   "description": "faunadb shell",
-  "version": "0.12.5",
+  "version": "0.12.6",
```

Comment:
The most recent release was 0.12.4: https://www.npmjs.com/package/fauna-shell

We should probably use 0.12.5 as the next release. If we go with 0.12.6, some users will wonder what happened to 0.12.5.

Reply:
sure, I'll close this PR then and release the latest version that is already in main branch